### DISCUSSION TITLE
Can i get the voice wave visualiser along with "Deepgram Aura" is it part of API?

### DISCUSSION THREAD

**Discussion by edhogithub** [deepgram/community#878] [General help]
Hi,

I would like to get the voice wave visualiser along with "Deepgram Aura" is it part of API?

Please advise how to add that in the AI chatbot.

Thanks

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment]
@edhogithub could you point me to what you are thinking about here? What do you mean by the "voice wave visualizer" ?
